ios/BlurhashViewManager.swift

```diff
@@ -9,49 +9,139 @@
 import Foundation
 import UIKit
 
+final class BlurhashViewWrapper: UIView, BlurhashViewDelegate {
+    private var blurhashView: BlurhashView
+
+    @objc var blurhash: NSString? {
+        get {
+            blurhashView.blurhash as NSString?
+        }
+        set {
+            blurhashView.blurhash = newValue as String?
+        }
+    }
+
+    @objc var decodeWidth: NSNumber {
+        get {
+            blurhashView.decodeWidth as NSNumber
+        }
+        set {
+            blurhashView.decodeWidth = newValue.intValue
+        }
+    }
+
+    @objc var decodeHeight: NSNumber {
+        get {
+            blurhashView.decodeHeight as NSNumber
+        }
+        set {
+            blurhashView.decodeHeight = newValue.intValue
+        }
+    }
+
+    @objc var decodePunch: NSNumber {
+        get {
+            blurhashView.decodePunch as NSNumber
+        }
+        set {
+            blurhashView.decodePunch = newValue.floatValue
+        }
+    }
+
+    @objc var decodeAsync: Bool {
+        get {
+            blurhashView.decodeAsync
+        }
+        set {
+            blurhashView.decodeAsync = newValue
+        }
+    }
+
+    @objc var resizeMode: NSString {
+        get {
+            convertContentMode(resizeMode: blurhashView.contentMode)
```

Comment:
We could simply extend the `ContentMode` type to add a `jsValue` or `stringRepresentation` or something like that - but let's leave it like this for now.